### PR TITLE
Change tooltip background text color to white (TUIM-23).

### PR DESCRIPTION
**Pull request by cahrens**
Before:

image

After:

image
image
image
image

### CODE REVIEW

**Comment by cahrens** [comment]
Integration test failure is unrelated.

File: src/components/TooltipTrigger.js [hunk 1]
```diff
@@ -59,10 +63,10 @@ const Tooltip = ({ side = 'bottom', type, target: targetId, children, id, delay
       (target.top + target.bottom) / 2 - position.top
     )
     return Utils.switchCase(finalSide,
-      ['top', () => ({ bottom: 0, left, transform: 'rotate(180deg)' })],
-      ['bottom', () => ({ top: 0, left })],
-      ['left', () => ({ right: 0, top, transform: 'rotate(90deg)' })],
-      ['right', () => ({ left: 0, top, transform: 'rotate(270deg)' })]
+      ['top', () => ({ bottom: 1, left, transform: 'rotate(180deg)' })],
```
Comment (cahrens) [inline src/components/TooltipTrigger.js:67]: The 1 is to avoid a faint line along the base of the triangle (where it meets up with the tooltip rectangle).

**Review by nawatts — APPROVED** [review]

**Comment by cahrens** [comment]
Slightly changed the box shadow based on UX feedback.

image

image

**Review by n8landolt — APPROVED** [review]
Great job - I didn't know about that 'filter' property for CSS!

File: src/components/TooltipTrigger.js (hunk 1 above)
Comment (msilva-broad) [reply src/components/TooltipTrigger.js:67]: nit: can we add a comment to the css block to explain the 1?

Comment (cahrens) [reply src/components/TooltipTrigger.js:67]: Done.

**Review by msilva-broad — APPROVED** [review]
looks great!

**Pull request merged**